app/views/layouts/_header.html.erb

```diff
@@ -169,6 +169,13 @@
           if($('#chat').is(':visible') == true)
             c.hide() ;
       });
+      $(window).on('keypress', function(e){
```

Comment:
This is awesome! Could we move this to a new file at `app/assets/javascripts/keybindings.js` and include it into `application.js`? You may have to wrap it in something like this so it loads after jQuery:

```js
$.ready(function() {
  // your code here
});
```

Thank you!!!!

Reply:
Hi @jywarren I've updated the PR as requested. :)
Thank You !!!!